app/templates/errors/400.html

```diff
@@ -0,0 +1,20 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Bad request - Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+<div class="error-page">
+    <header class="page-heading-smaller">
+      <h1>Sorry, there was a problem with your request</h1>
+    </header>
+    <p>
+      {% if error_message %}
+        {{ error_message }}
+      {% else %}
+        Please try again.
```

Comment:
This is probably bad content (I know/suspect you copied this from user-frontend, so 🤷‍♂️ ).

The buyer-frontend 400 template says "Please do not attempt the same request again.", which is more sensible - as a bad request shouldn't be repeated without changing something in the request.

Really should centralise these error templates one day...

Reply:
Done and done. Will stick a PR in for the user-frontend too.